WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -238,10 +238,10 @@ def post_results
                 end
                 record_strs
               end.flatten
-              "#{uniqueName} #{record_strs.join(", ")}"
+              "#{uniqueName} #{record_strs.join(', ')}"
```

Comment:
I think there was a performance penalty for using double quoted strings compared to single quoted strings because they allow for interpolation, but this has been fixed in recent Ruby versions or at least the difference in performance isn't significant anymore: http://stackoverflow.com/questions/1836467/is-there-a-performance-gain-in-using-single-quotes-vs-double-quotes-in-ruby

👍 on ignoring this rule. Another disadvantage of using `''` everywhere is the added diff noise because you can neither use escape characters nor retroactively add interpolation without changing `'` to `"`.